lib/release/questions.js

```diff
@@ -11,34 +11,44 @@ function askQuestions(themeConfig, githubToken, remotes, callback) {
 
     const currentVersion = themeConfig.getVersion();
 
+    const nextPatchVersion = semver.inc(currentVersion, 'patch');
+    const nextMinorVersion = semver.inc(currentVersion, 'minor');
+    const nextMajorVersion = semver.inc(currentVersion, 'major');
+    const nextReleaseCandidate = currentVersion.includes('-rc.') ?
+      semver.inc(currentVersion, 'prerelease', 'rc') :
+      semver.inc(currentVersion, 'minor') + '-rc.1';
+
     const questions = [
         {
             name: 'version',
             type: 'list',
             message: 'What type of release would you like to do? ' + 'Current version: '.cyan + currentVersion,
             choices: [{
-                value: 'patch',
-                name: 'Patch:  '.yellow + semver.inc(currentVersion, 'patch').yellow + '   Backwards-compatible bug fixes.',
+                name: 'Release Candidate:  '.yellow + nextReleaseCandidate.yellow + '   Internal release for testing.',
+                value: nextReleaseCandidate,
             },
             {
-                value: 'minor',
-                name: 'Minor:  '.yellow + semver.inc(currentVersion, 'minor').yellow + '   Feature release or significant update.',
+                name: 'Patch:  '.yellow + nextPatchVersion.yellow + '   Backwards-compatible bug fixes.',
+                value: nextPatchVersion,
             },
             {
-                value: 'major',
-                name: 'Major:  '.yellow + semver.inc(currentVersion, 'major').yellow + '   Major change.',
+                name: 'Minor:  '.yellow + nextMinorVersion.yellow + '   Feature release or significant update.',
+                value: nextMinorVersion,
+            },
+            {
+                name: 'Major:  '.yellow + nextMajorVersion.yellow + '   Major change.',
+                value: nextMajorVersion,
             },
             {
-                value: 'custom',
                 name: 'Custom: ?.?.?'.yellow + '   Specify version...',
+                value: 'custom',
             }],
-            filter: bumpType => semver.inc(currentVersion, bumpType),
         },
         {
             name: 'version',
             type: 'input',
             message: 'What specific version would you like',
-            when: answers => !answers['version'],
+            when: answers => answers['version'] == 'custom',
```

Comment:
does this still work if nothing is selected?

Reply:
The only choices available for version are the well defined ones or `custom`